data/json/items/comestibles.json

```diff
@@ -14659,6 +14659,33 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "fresh_fries",
+        "name" : "French fries",
+        "name_plural" : "French fries",
+        "weight" : 115,
+        "color" : "yellow",
+        "addiction_type" : "none",
+        "spoils_in" : 72,
+        "stim" : 0,
+        "container" : "box_small",
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : -1,
+        "healthy" : -1,
+        "addiction_potential" : 0,
+        "nutrition" : 3,
```

Comment:
I know they're junk food and all, but this is way lower than what it ought to be.